lib/media/drm_engine.js

```diff
@@ -1445,6 +1504,33 @@ shaka.media.DrmEngine = class {
     this.playerInterface_.onKeyStatus(shaka.util.MapUtils.asObject(publicMap));
   }
 
+  /**
+   *
+   * @param {string} keySystem
+   *
+   * @private
+   */
+  static isPlayReadyKeySystem_(keySystem) {
+    return keySystem == 'com.microsoft.playready.recommendation' ||
+        keySystem == 'com.microsoft.playready';
+  }
+
+  /**
+   * We may change the key system internally, but we don't want developers
+   * to add other key systems into their configuration.
+   *
+   * @param {string} keySystem
+   *
+   * @private
+   */
+  static getConfigKeySystem_(keySystem) {
```

Comment:
How about just changing the `keySystem` function to do this?

Reply:
I will change it to use this one, because some times we only want to check the keySystem string, outside of a DrmInfo.
The keySystem function provides useful check on the parameter existence and I think we don't want to repeat that check each time we call it.